docs/appendix.rst

```diff
@@ -524,69 +509,104 @@ To fix this, install the Python "wheel" package::
-OpenSSL and the WBEM server do not agree about which SSL/TLS protocol level to
-use.
+This is an error that is created by the SSL library (normally OpenSSL or
+LibreSSL) and handed back up to the SSL module of Python which hands it up to
+pywbem. The error indicates that OpenSSL and the WBEM server do not agree about
+which SSL/TLS protocol level to use.
 
 Pywbem specifies SSL parameters such that the highest SSL/TLS protocol version
-is used that both the client and server support. Thus, pywbem does not put any
-additional restrictions on top of OpenSSL.
+is used that both the client and server support. Pywbem itself does not put any
+additional TLS version restrictions on top of SSL library.
 
 Debian buster includes OpenSSL 1.1.1d and increased its security settings to
 require at least TLS 1.2 (see https://stackoverflow.com/a/53065682/1424462).
 
-This error means that the WBEM server side does not yet support
-TLS 1.2 or higher or that the SSL library used by pywbem does not support
-TL 1.2.
-
-This issue can be corrected by:
+This issue can possibly be corrected by:
 
 1. If the current version of urllib3 is less than 2.0, update the version
-   of urllib3 (ex. ``pip install --upgrade  --upgrade-strategy eager urllib3``). Since
-   the pywbem install does not force an eager update of packages, if a valid
-   previous version of urllib3 exists it will not be upgraded to 2.0+ in the
-   reinstallation of pywbem.
+   of urllib3 (ex. ``pip install --upgrade  --upgrade-strategy eager urllib3``).
+   Since the pywbem install does not force an eager update of packages, if a
+   valid previous version of urllib3 exists it will not be upgraded to 2.0+ in
+   the reinstallation of pywbem.
 
 2. If the current version of urllib3 is greater than 2.0, the previous version
    of urllib3 (ex. version 1.26.5) can be installed (ex. ``pip install
    urllib3<2.0``)
 
 3. Adding TLS 1.2 support to the server side (preferred) or by lowering the
-    minimum TLS level OpenSSL requires on the client
-    side (which lowers security). The latter can be done by changing the
-    ``MinProtocol`` parameter in the OpenSSL config file on the client OS
-    (typically ``/etc/ssl/openssl.cnf`` on Linux and OS-X,
-    and ``C:\OpenSSL-Win64\openssl.cnf`` on Windows).
-    At the end of the file there is::
+   minimum TLS level required on the client side (which lowers security). With
+   OpenSSL The latter can be done by changing the ``MinProtocol`` parameter in
+   the OpenSSL config file on the client OS (typically ``/etc/ssl/openssl.cnf``
+   on Linux and OS-X, and ``C:\OpenSSL-Win64\openssl.cnf`` on Windows).
+
+   At the end of the file there is::
 
         [system_default_sect]
         MinProtocol = TLSv1.2
         CipherString = DEFAULT@SECLEVEL=2
 
+4. If the issue is the use of LibreSSL as the OS level SSL implementation, the
+   issue may be the LibreSSL version or the urllib3 version as versions of
+   urllib3 less than 2.0.3 did not support libreSSL.
```

Comment:
I needed to read this a few times to understand what it means.

Suggestion: "If LibreSSL is used as the OS level SSL implementation, the issue may be that urllib3 before version 2.0.3 did not support LibreSSL."

If there is an additional requirement on LibreSSL versions to be used, we should state that separately (and with the explicit versions supported).

Reply:
I added a section based on the ImportError urlliber ... issue.